clang/lib/AST/ASTStructuralEquivalence.cpp

```diff
@@ -1295,6 +1306,21 @@ static bool IsStructurallyEquivalent(StructuralEquivalenceContext &Context,
   return true;
 }
 
+static bool IsStructurallyEquivalent(StructuralEquivalenceContext &Context,
```

Comment:
Should we also be checking the storage class and the various bits for example `CXXForRangeDecl` etc?

Reply:
Thanks for your remind. Storage checking is added and other various bits can be ignored(I think, or at present), because these bits is added in `Sema` and constructor doesn't contain these bits. On the other hand, when import `VarDecl` we also ignore these bits.